docs/content/commands/npm-init.md

```diff
@@ -253,6 +253,17 @@ This value is not exported to the environment for child processes.
 <!-- automatically generated, do not edit manually -->
 <!-- see lib/utils/config/definitions.js -->
 
+#### `workspaces-update`
+
+* Default: true
+* Type: Boolean
+
+If set to true, the npm cli will run an update after operations that may
```

Comment:
what does "run an update" mean?

Reply:
it means the npm cli will run a `npm update <workspace-name>` cmd behind the scenes, to be clear this text is already there, ref: https://docs.npmjs.com/cli/v8/using-npm/config#workspaces-update - in case you want to propose improvements here, I'm due to a follow up PR improving docs and would appreciate the feedback 😊

Reply:
sure, but it doesn't seem clear that "run an update" means this. Perhaps:
```suggestion
If set to true, the npm cli will run an update (`npm update <workspace-name>`) after operations that may
```
?

Reply:
I think that's very close to what I want, I'll follow up with a docs PR once this lands 😊 thanks!